browser/tor_window_search_engine_provider_controller.cc

```diff
@@ -18,14 +19,13 @@ TorWindowSearchEngineProviderController(Profile* profile)
 
   alternative_search_engine_provider_in_tor_.Init(
       kAlternativeSearchEngineProviderInTor,
-      otr_profile_->GetOriginalProfile()->GetPrefs());
+      profile->GetOriginalProfile()->GetPrefs());
```

Comment:
can you explain why GetOriginalProfile from regular profile? That will just return itself

Reply:
|profile| is off the record profile.

Reply:
I see. I just checked the caller.